test/lib/agent_helper.js

```diff
@@ -250,9 +250,15 @@ helper.unloadAgent = (agent, shimmer = require('../../lib/shimmer')) => {
 
 helper.loadTestAgent = (t, conf, setState = true) => {
```

Comment:
We have so many of these methods that do almost the same thing. It's very confusing to know which one to use.

Reply:
agreed. I didn't know this one even existed until these tests.

Reply:
this was is really only used when you a root test that needs an agent